typed-racket-test/fail/cast-mod1.rkt

```diff
@@ -1,5 +1,5 @@
 #;
-(exn-pred exn:fail:contract? #rx".*produced: 3" #rx".*promised: String.*" )
+(exn-pred exn:fail:contract? #rx".*produced: 3" #rx".*promised: String.*" #rx"6\\.0")
```

Comment:
That's the line/column number. It's testing that the source location is intact. (it was intact for this case before too, but there are some situations in which it's missing)